### ISSUE TITLE
How to get model information ? (model summary in keras)

### ISSUE THREAD

**Issue by hongym7** [tensorflow/models#10054]
In keras 

```
from keras.models import Sequential
from keras.layers import Dense
import numpy as np
 
x = np.array([1,2,3,4,5])
y = np.array([1,2,3,4,5])
x2 = np.array([11,12,13,14,15])
 
model = Sequential()
model.add(Dense(5, input_dim=1, activation='relu'))
model.add(Dense(3))
model.add(Dense(1))
 
model.summary()
```

How to get model information in Tensorflow 2 Object Detection API ? 

```
loaded = tf.saved_model.load('XXXXXXX/saved_model')  
get model summay .....
```

**Comment by jvishnuvardhan** [comment]
@hongym7 Only keras model has `summary` object. If you want to convert a TF saved model into Keras, please follow the approach mentioned [here](https://github.com/tensorflow/tensorflow/issues/42425#issuecomment-726921556). Once you convert the TF model into Keras model, then you can access `summary` object as in your example. Thanks!

**Comment by hongym7** [comment]
> @hongym7 Only keras model has `summary` object. If you want to convert a TF saved model into Keras, please follow the approach mentioned [here](https://github.com/tensorflow/tensorflow/issues/42425#issuecomment-726921556). Once you convert the TF model into Keras model, then you can access `summary` object as in your example. Thanks!

Thank you

**Comment by kabyanil** [comment]
I hope this feature to check model summary gets added in the API someday. It's really useful as it displays all the layers in use. I'm finding it quite hard to get the model summary as the API masks all the implementation.
